### PR TITLE
Feature/jira 045 view food search results

### PR DESCRIPTION
**Pull request by elijahbrooks**

### CODE REVIEW

File: src/main/java/com/crumbs/fss/controller/MainController.java [hunk 1]
```diff
@@ -41,7 +41,7 @@
         Page<Restaurant> restaurants = restaurantSearchService.getRestaurants(pageRequest)
                 .orElseThrow();
         if (filter != null && filter.length > 0)
-            restaurants = restaurantSearchService.filterResults(filter, pageRequest);
+            restaurants = restaurantSearchService.filterRestaurantResults(null, filter, pageRequest);
```
Comment (elijahbrooks) [inline src/main/java/com/crumbs/fss/controller/MainController.java:44]: Passing null as an argument, is this okay?

File: src/main/java/com/crumbs/fss/controller/MainController.java [hunk 2]
```diff
@@ -54,39 +54,48 @@
...
-        Page<MenuItem> menuItems = restaurantSearchService.getMenuItems(pageRequest)
+        Page<MenuItem> menuItems = restaurantSearchService.getMenuItems(pageRequest, restaurantId)
                 .orElseThrow();
 
         return new ResponseEntity<>(menuItems, HttpStatus.OK);
     }
 
-    @GetMapping("/menuitems/search")
-    public ResponseEntity<Page<MenuItem>> getMenuItems(
+    @GetMapping("/restaurants/menuitems")
```
Comment (jim-brower) [inline src/main/java/com/crumbs/fss/controller/MainController.java:81]: Just a thought, maybe this should just be /menuitems ?

File: src/main/java/com/crumbs/fss/controller/MainController.java (hunk 1 above)
Comment (jim-brower) [reply src/main/java/com/crumbs/fss/controller/MainController.java:44]: I think this is ok

**Review by jim-brower — APPROVED** [review]
looks good, I don't think you need to change anything

**Review by JonathanFrey0000 — APPROVED** [review]
Looks good

**Pull request merged**